### ISSUE TITLE
HTML validation issues on `WWW`

### ISSUE THREAD

**Issue by Febakke** [digdir/designsystemet#4578]
### Description of the bug

Retested Parsing when updating our Accessibility declaration and found some new errors. We should take a look if they are causing any issues and should be fixed. 
<img width="1402" height="775" alt="Image" src="https://github.com/user-attachments/assets/f1c5dda8-351f-401a-a6e2-9ec21b63c77e" />

- [ ] Test on component pages
- [ ] Test on pattern pages
- [ ] Test on documentation

### Steps To Reproduce

Use a HTML validator on our pages.
